frontend/src/pages/Dashboard.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <ContentLayout title="Dashboard">
+        dashboard text
+    </ContentLayout>
+</template>
+<script>
+    import ContentLayout from "../components/layout/ContentLayout";
```

Comment:
import ContentLayout from '@/components/layout/ContentLayout.vue';
It might be better to use the full path to the component, so it will always remain correct when you move this file

Reply:
didn't notice that, autocomplete)